src/cow-react/modules/limitOrders/pure/OrderType/index.tsx

```diff
@@ -45,61 +43,25 @@ function OrderTypePicker({ isPartiallyFillable, partiallyFillableOverride }: Ord
 
   return (
     <Menu>
-      {({ isExpanded }) => (
+      {({ isExpanded }: { isExpanded: any }) => (
         <>
-          <StyledMenuButton>
-            <span>{labelText}</span>
-            <SVG src={IMAGE_CARET_DOWN} description="dropdown icon" className={isExpanded ? 'expanded' : ''} />
-          </StyledMenuButton>
-          <StyledMenuList portal={true}>
-            <StyledMenuItem onSelect={onSelect}>{dropDownText}</StyledMenuItem>
-          </StyledMenuList>
+          <styledEl.Wrapper>
+            <styledEl.StyledMenuButton>
+              <styledEl.LabelText>{labelText}</styledEl.LabelText>
+              <styledEl.StyledSVG
+                src={IMAGE_CARET_DOWN}
+                description="dropdown icon"
+                className={isExpanded ? 'expanded' : ''}
+              />
+            </styledEl.StyledMenuButton>
+
+            <styledEl.StyledMenuList portal={false}>
+              <styledEl.StyledMenuHeader>Select order type</styledEl.StyledMenuHeader>
```

Comment:
Not sure we need a header.
It's just a single element in the list, and I assume it's self explanatory.

Reply:
Same thought

Reply:
Ok, will remove it then